src/modules/referral/services/queries.ts

```diff
@@ -17,6 +17,15 @@ export const getReferralsQuery = () => {
   `;
 };
 
+export const getReferralsTotalQuery = () => {
+  return `
+    select count(*)
+    from deposits_mv as d
+    where d."referralAddress" = $1 or
```

Comment:
So you're saying either a referral has a referral address or a depositor address?

Reply:
@nicholaspai 
yes. because you get 25% if the address is the depositor address

Reply:
$1 is a particular address that's being checked